2.cellprofiler_analysis/scripts/nf1_analysis.py

```diff
@@ -86,3 +88,4 @@
         sqlite_name=plate,
```

Comment:
```
# run through each plate with each set of paths based on dictionary
for plate, info in plates_info_dictionary.items():
    path_to_pipeline = info["path_to_pipeline"]
    path_to_images = info["path_to_images"]
    print(f"Running analysis on {plate}!")
    # run analysis pipeline
    cp_utils.run_cellprofiler(
        path_to_pipeline=path_to_pipeline,
        path_to_output=path_to_output,
        path_to_images=path_to_images,
        # name each SQLite file after plate name
```

May also consider adding the `path_to_output.mkdir(exist_ok=True)` here like you did in the 1.cellprofiler_ic/nf1_ic.ipynb file

Reply:
I agree, so I added this in the code cell above instead and edited the documentation.